### PR TITLE
add About Us section

### PR DESCRIPTION
**Pull request by agonzalez0515**
Will probably want to add more information, just wanted something to exist.

Issue #85

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
 
+### 👩‍💻💕About Us
+
+Angie and Arlene are LA natives that met while attending Dev Bootcamp in San Francisco.  After bootcamp was over and they were back in LA, they wanted to once again be part of an amazing, welcoming community like DBC was. They found Hoodie through Rails Girls! This project is extra special for them as it is their first contribution to open source.
```
Comment (gr2m) [inline README.md]: maybe instead of Rails Girls, say Rails Girls Summer of Code and link to their website? https://railsgirlssummerofcode.org/

Comment (Techforchange) [reply README.md]: @agonzalez0515 Would it be too much if we add our pictures like this? 
image

Comment (Techforchange) [reply README.md]: I can add that once I merge the PR.  Do you mind adding @gr2m 's suggestion so I can merge.

**Comment by agonzalez0515** [comment]
@Techforchange added link to rails girls.

**Pull request merged**